lib/identifiers/doi.rb

```diff
@@ -22,9 +22,9 @@ def self.extract(str)
       str
         .scan(PATTERN)
         .map { |doi|
-          next doi.downcase if doi !~ /\p{Punct}\z/ || doi =~ /\(.+\)\z/
+          next doi.downcase if doi !~ /\p{Punct}\z/ || doi =~ /\(.+\)\z/ || doi =~ /2-#\z/
```

Comment:
I wonder if there's virtue in naming this new regex as a constant (e.g., `WILEY_SUFFIX`), so that the code is more self-describing?